karma.conf.js

```diff
@@ -44,6 +44,7 @@ module.exports = function(config) {
             {pattern: 'warp/**/*.js', included: false},
             {pattern: 'platform/**/*.html', included: false},
             {pattern: 'src/**/*.html', included: false},
+            {pattern: 'src/**/*.json', included: false},
```

Comment:
Should we just say `src/**/*` so that we don't need to do this for every new file type? Would this result in undesired behavior in some way?